ETL/reviews_photos.js

```diff
@@ -0,0 +1,7 @@
+const transform = require('./transform');
+
+const numbers = new Set(['id', 'review_id']);
+const nVals = new Set();
+const field = ['id', 'review_id', 'url'];
+
+transform('reviews_photos', numbers, nVals, field);
```

Comment:
This is what I meant with my comment about the field argument. You include an argument for field here, but don't have any use for it when you define the transform function.

Reply:
@wooseok0717 do you see what I mean?